src/models/NFTAttributes.ts

```diff
@@ -1,4 +1,4 @@
-import { RoyaltyAttributes } from '../nevermined/Assets'
+import { RoyaltyAttributes } from '../nevermined/api/AssetsApi'
 import BigNumber from '../utils/BigNumber'
 
 export enum ERCType {
```

Comment:
We need to find a better way to do this. All these types and enums are very confusing